test/testWorkspace/src/Area.flix

```diff
@@ -1,8 +1,11 @@
 
 /// Computes the area of the given shape using
 /// pattern matching and basic arithmetic.
-def area(s: Shape): Int32 = match s {
-    case Shape.Circle(r)       => 3 * (r * r)
-    case Shape.Square(w)       => w * w
-    case Shape.Rectangle(h, w) => h * w
+def area(s: Shape): Int32 = {
```

Comment:
Should we not try to keep test cases more separate/independent or?

Reply:
We can for the future, but I think right now it would be a waste of time to find a new example and recompile the binary files.

Reply:
Ok